src/lib/graphqlQueries.ts

```diff
@@ -0,0 +1,112 @@
+export const queryCategoryProducts = `
```

Comment:
I think it might be better to have their own files per query for maintainability